docs/source/private_data_tutorial.rst

```diff
@@ -119,58 +142,70 @@ Read and Write private data using chaincode APIs
 ------------------------------------------------
 
 The next step in understanding how to privatize data on a channel is to build
-the data definition in the chaincode. The marbles private data sample divides
-the private data into two separate data definitions according to how the data will
+the data definition in the chaincode. The asset tranfer private data sample divides
+the private data into three separate data definitions according to how the data will
 be accessed.
 
 .. code-block:: GO
 
  // Peers in Org1 and Org2 will have this private data in a side database
- type marble struct {
-   ObjectType string `json:"docType"`
-   Name       string `json:"name"`
-   Color      string `json:"color"`
-   Size       int    `json:"size"`
-   Owner      string `json:"owner"`
+ type Asset struct {
+	Type  string `json:"objectType"` //Type is used to distinguish the various types of objects in state database
+	ID    string `json:"assetID"`
+	Color string `json:"color"`
+	Size  int    `json:"size"`
+	Owner string `json:"owner"`
  }
 
+ // AssetPrivateDetails describes details that are private to owners
+
  // Only peers in Org1 will have this private data in a side database
- type marblePrivateDetails struct {
-   ObjectType string `json:"docType"`
-   Name       string `json:"name"`
-   Price      int    `json:"price"`
+ type AssetPrivateDetails struct {
+	ID             string `json:"assetID"`
+	AppraisedValue int    `json:"appraisedValue"`
  }
 
-Specifically access to the private data will be restricted as follows:
+ // Only peers in Org2 will have this private data in a side database
+ type AssetPrivateDetails struct {
+	ID             string `json:"assetID"`
+	AppraisedValue int    `json:"appraisedValue"`
+ }
+
+Specifically, access to the private data will be restricted as follows:
+
+- ``ID, color, size, and owner`` will be visible to all members of the channel defined in the collection policy (Org1 and Org2).
+- ``AppraisedValue`` The record of Org1's opinion of appraised value is only visible to members of Org1 in their collection.
+- ``AppraisedValue`` The record of Org2's opinion of appraised value is only visible to members of Org2 in their collection.
```

Comment:
repetition? The same field is listed twice

Reply:
No, if you view in the code snippet you will see it referenced to each org collection for context.